bookkeeper-slogger/slf4j/build.gradle

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+plugins {
+    id 'java'
+}
+
+dependencies {
+    implementation project(':bookkeeper-slogger:api')
+    implementation depLibs.slf4j
+    testImplementation depLibs.slf4jSimple
+    testImplementation depLibs.junit
+}
+
+jar.archiveBaseName = 'bookkeeper-slogger-slf4j'
```

Comment:
publish jar name should be start as `org.apache.bookkeeper-`

Reply:
i only overrided the name to avoid getting tripped up by the log4shell owasp regex.  in general, we don't include `org.apache.bookkeeper-` in the jar file names, or even bother to rename rather generic ones like `stream/clients/java/kv/build/libs/kv.jar`

```
ephemeral ~/src/bookkeeper-directio |> ls **/*.jar | wc -l
      40
ephemeral ~/src/bookkeeper-directio |> ls **/*.jar | fgrep org.apache
```